cmd/kev/cmd/init.go

```diff
@@ -118,25 +119,43 @@ func runInitCmd(cmd *cobra.Command, _ []string) error {
 	if skaffold {
 		skaffoldManifestPath := path.Join(workingDirAbs, kev.SkaffoldFileName)
 
-		skaffoldManifest, err := kev.PrepareForSkaffold(manifest, skaffoldManifestPath, envs)
-		if err != nil {
-			return displayInitError(err)
-		}
+		// set skaffold path in kev manifest
+		manifest.Skaffold = kev.SkaffoldFileName
 
-		// don't override existing skaffold.yaml
 		if manifestExistsForPath(skaffoldManifestPath) {
```

Comment:
So this is a `createOrUpdateSkaffoldManifest` type setup. Refactoring it into a `func` will help make it more obvious.